### PR TITLE
Consolidated changes

### LINKED ISSUE #7: Consolidate Changes
**Bugs to be fixed**
- Bug: Restore note in Trash page moves it to archive page
- Bug: Accepts note with empty Title
- Sidebar Does not close after navigating to Dashboard page
- Shows empty space in Dashboard at the place of graph when user has not added any note

**CSS Issue**
- Add Disabled property for button

**Other changes**
- Work on Edit Note validations

### PR DESCRIPTION
**Pull request by ttahm3d**
**Bugs to be fixed**

- Bug: Restore note in Trash page moves it to archive page
- Bug: Accepts note with empty Title
- Sidebar Does not close after navigating to Dashboard page
- Shows empty space in Dashboard at the place of graph when user has not added any note

**CSS Issue**

- Add Disabled property for button

**Other changes**

- Work on Edit Note validations

Closes bug #7

### CODE REVIEW

**Review by MonikaJhingan — APPROVED** [review]
The folder structure and the  code is implemented very well. Indentation is also done. All the precise features have been implemented nicely. Needs to be merged. Few things are new that anyone can learn from this project.

**Review by Gautam2305 — APPROVED** [review]
everything looks clean and good to merge.
Some things i liked about this PR:
- loved how you described your work in readme file in detail
- got to know about semantic tag <Pie> 
- code was well written and structured, was consistent too

File: README.md [hunk 1]
```diff
@@ -1 +1,72 @@
...
+
+- add notes
+  - tag them under various available categories
+    - Personal
+    - Learning
+    - Work
+    - To do
+    - In Progress
+    - Completed
+- Can view a breif summary of their notes in the dashboard
```
Comment (coldpigli) [inline README.md]: A small typo here. Should be 'brief'.

File: public/index.html [hunk 2]
```diff
@@ -5,7 +5,10 @@
     <link rel="icon" href="%PUBLIC_URL%/NoteItFavicon.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta name="description" content="Notes App" />
+    <meta
+      name="description"
+      content="Notes App build with React and Supabase backend"
```
Comment (coldpigli) [inline public/index.html]: Typo: Built

**Review by coldpigli — APPROVED** [review]
Found a couple of typos you might wanna fix.
Everything else looks great.

**Review by omkarpatke — APPROVED** [review]
Everything looks really good👍
Appreciation:
 - Clean and readable code
 - file structure is very good.
 - variable and function naming is also good.

**Review by omkarpatke — APPROVED** [review]
Everything looks really good👍
Appreciation:
 - Clean and readable code
 - file structure is very good.
 - variable and function naming is also good.

**Pull request merged**